docs/user/gcp/install_upi.md

```diff
@@ -78,6 +78,21 @@ If you do not want the cluster to provision compute machines, remove the compute
 rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ```
 
+### Make control-plane nodes unschedulable
+
+Currently [emptying the compute pools](#empty-compute-pools) makes control-plane nodes schedulable.
+But due to a [Kubernetes limitation][kubernetes-service-load-balancers-exclude-masters], router pods running on control-plane nodes will not be reachable by the ingress load balancer.
+Update the scheduler configuration to keep router pods and other workloads off the control-plane nodes:
+
+```sh
+python -c '
+import yaml;
+path = "manifests/cluster-scheduler-02-config.yml"
+data = yaml.load(open(path));
+data["spec"]["mastersSchedulable"] = True;
```

Comment:
Same as https://github.com/openshift/installer/pull/2440/files#r330741656

Reply:
Fixed in 55caef592 -> 485057af6.